### ISSUE TITLE
Dodanie źródła czeskiego

### ISSUE THREAD

**Issue by kopernik1elfka** [postscripter8/czeska_ustawa_o_broni#15]
Mam wrażenie, że niektóre ustępy nie są dokładnie przetłumaczone na angielski i łatwiej byłoby je zrozumieć, jeśli je przetłumaczyć bezpośrednio z czeskiego (czeski ma podobną gramatykę, układ zdań itd.).

Może warto byłoby wstawić do kodu oryginał ustawy (w jednym pliku, bez rozdzielania na artykuły)?

**Comment by just-browsin** [comment]
Tu jest link do oryginału : https://www.zakonyprolidi.cz/cs/2002-119#cast1 , może się przydać w przyszłości.

**Comment by postscripter8** [comment]
Tak zrobimy. Po pracy wrzucę do źródeł.

**Comment by kopernik1elfka** [comment]
Mam tylko wrażenie, że nie jest to dokładnie ten akt, który tłumaczymy. Kawałek, który dziś oglądałam (Artykuł 57) kompletnie inaczej brzmi w tej wersji niż w angielskiej. Nawet inaczej wygląda, ma podpunkty, inny tytuł i takie pierdoły.
Chyba @just-browsin podaje link do tekstu ogłoszonego, a my pracujemy nad tekstem ujednoliconym (albo jednolitym, nie wiem, nie znam się), albo na odwrót. W pierwszych słowach angielskiej wersji jest o tym, że zawiera zmiany wprowadzone w innych aktach (ustawach?).
Udało mi się znaleźć [tutaj](http://www.mvcr.cz/soubor/zbrane-6-8-2016-pdf.aspx) (UWAGA! Od razu się pdf otwiera), ale właśnie artykuł o którym mówię jest wykreślony i zastąpiony przez ten, do którego linka podaje @just-browsin. To tak, jakbyśmy pracowali nad przestarzałą wersją tego aktu...

**Comment by just-browsin** [comment]
Tak by wyglądało - w art. 16 tłumaczenie angielskie mówi o 5 latach ważności pozwolenia na broń, a to z mojego linku ma jak byk 10 lat ważności. Czy mamy jak potwierdzić, że tłumaczenie ang. jest w miarę aktualne i nadaje się do dalszej pracy?

**Comment by kopernik1elfka** [comment]
Poprawcie mnie, jeśli źle kminię. Na stronie [ministerstvo-vnitra-ceske-republiky](http://www.mvcr.cz/ministerstvo-vnitra-ceske-republiky.aspx), która jest odpowiednikiem naszego MSWiA jest zakładka prawodawstwo(Legislativa). A w niej jest pierwsza zakładka ze zbiorem ustaw i traktatów międzynarodowych (Sbírka zákonů a Sbírka mezinárodních smluv) - to jest chyba odpowiednik naszego ISAPu. Tam jak się wpisze "119/2002" (w pierwszy textbox) i wciśnie szukaj (Vychledat) to oczywiście zwraca wersję opublikowaną 8 marca(?) 2002 roku - pierwotną - w której jest wyszczególnione, że zmienia m. in ustawę 156/2000. Natomiast jak się wpisze "zákon o zbraních" (ustawa o broni) to nam wyskakuje kilka aktów: m. in jako pierwszy 229/2016, który w sposób znaczący zmienia tą ustawę 119, tak, że brzmi ona w ten sposób, co [tutaj](https://www.zakonyprolidi.cz/cs/2002-119#p56), [tutaj](https://portal.gov.cz/app/zakony/zakonPar.jsp?idBiblio=53276&nr=119~2F2002&rpp=15#local-content) i [tutaj](http://www.mvcr.cz/soubor/zbrane-6-8-2016-pdf.aspx)(OTWIERA SIĘ PDF), z tym, że w nagłówku jest 'ze zmianami wprowadzonymi przez ustawę z 6. 8. 2016' (s vyznačením změn účinných od 6. 8. 2016). Także, ciemno to widzę.
@postscripter8 - skąd wziąłeś tekst angielski ustawy?

**Comment by kopernik1elfka** [comment]
Nie czekając na odpowiedź @postscripter8 (lub ją częściowo mając), postanowiłam przetłumaczyć artykuł 56 z najaktualniejszej czeskiej wersji ustawy (czyli z tego pdfa, co podawałam wcześniej). Na serwerze w źródłach jest poprzednie tłumaczenie z wersji angielskiej, więc w razie czego można do niej powrócić.

**Comment by kopernik1elfka** [comment]
@just-browsin Przepraszam, że dodałam do źródła art. 16  - tłumaczenie kogoś innego. Wiem, że nad tym pracowałeś... ale tekst już był.

**Comment by postscripter8** [comment]
@kopernik1elfka @just-browsin Macie rację. W istocie pracujemy nad tekstem nie do końca aktualnym. Niestety treść aktualnie obowiązujących przepisów jest niedostępna w języku angielskim.

[Treść nad którą pracujemy](http://www.mvcr.cz/mvcren/file/zakon-o-strelnych-zbranich-a-strelivu-en-doc.aspx) jest treścią ujednoliconą po wprowadzeniu poprawki [484/2008](https://www.zakonyprolidi.cz/cs/2008-484), pobraną ze strony [mvcr.cz](http://www.mvcr.cz/mvcren/article/firearms-and-ammunition-legal-framework.aspx).
Niestety nie jest to najnowsza treść dostępna w języku angielskim. [Najnowsza dostępna w jęz. angielskim](http://www.mvcr.cz/mvcren/file/zakon-o-zbranich-en-prijate-zmeny-doc.aspx), oznaczona jako aktualna, jest treścią ujednoliconą po nowelizacji [167/2012](https://www.zakonyprolidi.cz/cs/2012-167#cast21).

Przejrzałem listę wszystkich nowelizacji ustawy od tego czasu. Wygląda na to, że **były trzy duże zmiany o zakresie kilkunastu/kilkudziesięciu artykułów**, pięć zmian o zakresie jednego-dwóch artykułów, a reszta to pojedyncze roszady słowne. Większość zmian, to dodanie jakiegoś słowa, lub zmiana sformułowania. Niestety są też zmiany znaczące.

Obawiam się, że objętość aktualnego tekstu jest za duża, aby podejmować się tłumaczenia go bezpośrednio z j. czeskiego na polski. Jednak objętość zmian w nowelach też jest znacząca.

Notabene [najnowsza zmiana](https://www.zakonyprolidi.cz/cs/2017-222#cast8) jest z przedwczoraj - 15.08.2017. Sama nowelizacja ogromna, ale w tłumaczonej ustawie jest tylko jedno zdanie zmienione, coś z EKB.

**Comment by just-browsin** [comment]
Czyli na dziś najlepsza metoda pracy byłaby taka, że kto da radę to pracuje bezpośrednio z czeskim tekstem, a pozostali muszą przynajmniej czeski tekst (i translator Google ;-) ) mieć otwarty i patrzeć, czy nie rzucają im się w oczy jakieś rozjazdy - liczby, liczby podpunktów, znaczne zmiany długości akapitów, takie rzeczy...
Chyba warto byłoby do readme dodać linka do aktualnego tekstu czeskiego.

**Comment by postscripter8** [comment]
Aktualny tekst czeski wygenerowany - w formie wydruku - przez stronę [zakonyprolidi.cz](https://www.zakonyprolidi.cz/cs/2002-119/zneni-20170815) w dodałem jako plik [formacie pdf](https://github.com/postscripter8/czeska_ustawa_o_broni/blob/master/zakonyprolidi_cs_2002_119_v20170815.pdf). Niech ktoś mnie sprawdzi, nie chcę drugi raz siary narobić :-(

**Comment by kopernik1elfka** [comment]
Zgadza się, sprawdziłam trzy ostatnie zmiany aktów i rzeczywiście te zmiany są odnotowane w tym pliku. Jedyny problem, że trzeba go pobrać i otworzyć jako pdf. (Po odświeżeniu, github go wyświetla)
W każdym razie, gdyby ktokolwiek miał problem z otworzeniem tego, prosím dać znać :-).

@postscripter8 - mógłbyś odnotować dodanie tego źródła w wiki (Tłumaczenie ustawy o broni i amunicji Republiki Czeskiej z języka angielskiego na polski - dodać podpunkt o źródle czeskim) i readme (Jakiej ustawy? - dodać link)?

**Comment by kopernik1elfka** [comment]
@just-browsin : chyba najwygodniej będzie otworzyć plik czeski(aktualny) i angielski - wkleić czeski do google translate, wkleić angielski, oba przetłumaczyć i porównać, czy się mniej więcej zgadzają. Bo poza tymi widocznymi, czasem są zmiany w kilku wyrazach. Postaram się sprawdzić (wieczorem lub w niedziele), które konkretnie zmieniają się diametralnie, a ile jest jednoliniowych. Jak coś, ja mogę tłumaczyć z czeskiego, to w miarę podobny język (trzeba tylko uważać, bo google często wyrazu z przedrostkiem ne nie neguje, a jest to ich zaprzeczenie), ale i tak fachowe określenia musicie Wy sprawdzić. Jak choćby te zapalky...
